Paper.tex

```diff
@@ -1116,7 +1116,7 @@ \subsection{Ommer Validation}
 \begin{equation}
 s(U, H) \equiv (P(H) = P(U)\; \wedge \; H \neq U \; \wedge \; U \notin B(H)_{\mathbf{U}})
 \end{equation}
-where $B(H)$ is the block of the corresponding header $H$.
+where $B_H$ and $P(H)$ are the block and the parent block of the corresponding header $H$ respectively.
```

Comment:
Is there some $B_H$ around here?

Reply:
@pirapira I'm sorry. My fault. I fixed it

Reply:
No worries.  I'm coming back today.